draft-ietf-dprive-dnsoquic.md

```diff
@@ -308,14 +308,30 @@ DOQ_NO_ERROR (0x00):
 : No error.  This is used when the connection or stream needs to be closed, but
   there is no error to signal.
 
-DOQ_INTERNAL_ERROR (0x01):
+DOQ_REQUEST_CANCELLED (0x01):
+: A DoQ client uses this to signal that it wants to cancel an 
+outstanding transaction.
+
+DOQ_INTERNAL_ERROR (0x02):
 : The DoQ implementation encountered an internal error and is incapable of
   pursuing the transaction or the connection.
 
```

Comment:
Why do you need to renumber the errors? Can't you just add this at the end of the list?

Reply:
I didn't think we had any of -03 that had implemented error codes... perhaps we should update the implementation section adding the draft version (e.g I _think_ Adguard_ are still on -02)

Reply:
I did implement 03 in quicdoq. Should have talked about it, I guess. In any case, the error codes 0 and 1 were in 02 already :-)